lambda/app/src/routes/client.ts

```diff
@@ -4,33 +4,19 @@ import { kebabCase } from 'lodash';
 import { fetchClient } from '../keycloak/client';
 
 export const getClient = async (session: Session, data: { requestId: number }) => {
-  try {
-    const request = await models.request.findOne({
-      where: {
-        idirUserid: session.idir_userid,
-        id: data.requestId,
-      },
-    });
+  const request = await models.request.findOne({
+    where: {
+      idirUserid: session.idir_userid,
+      id: data.requestId,
+    },
+  });
 
-    console.log(request.environments, request.projectName);
+  const proms = [];
+  request.environments.forEach((env) => {
+    proms.push(fetchClient({ environment: env, realmName: 'onestopauth', clientId: kebabCase(request.projectName) }));
+  });
 
-    const proms = [];
-    request.environments.forEach((env) => {
-      proms.push(fetchClient({ environment: env, realmName: 'onestopauth', clientId: kebabCase(request.projectName) }));
-    });
+  const clients = await Promise.all(proms);
```

Comment:
I think here we can just return the promise instead of awaiting and returning clients,
`return Promise.all(proms)`
since we dont have to worry about catching the error anymore

Reply:
right, I will update it